charts/rollups-validator-node/README.md

```diff
@@ -0,0 +1,276 @@
+# rollups-validator-node
+
+# Package for Cartesi Rollups validator nodes
+
+![Version: 0.5.1](https://img.shields.io/badge/Version-0.5.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.9.1](https://img.shields.io/badge/AppVersion-0.9.1-informational?style=flat-square)
+
+Refer to the [official documentation](https://docs.cartesi.io/cartesi-rollups/overview/) for more information about Cartesi Rollups.
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| endersonmaia |  |  |
+| oap75 |  |  |
+
```

Comment:
This is generated via template, maybe we need to improve the chart.yaml to add better maintainers information.